src/renderer/viz/expressions/ramp.js

```diff
@@ -143,7 +142,7 @@ export default class Ramp extends BaseExpression {
     }
 
     _getColorValue (texturePixels, m) {
-        const index = Math.round(m * MAX_BYTE_VALUE);
+        const index = !Number.isFinite(m) || Number.isNaN(m) ? 0 : Math.round(m * MAX_BYTE_VALUE);
```

Comment:
> !Number.isFinite(m) 

I guess that GLSL behavior for `Inf` (positive) is to get the last color, not the first one

Reply:
Does it mean that for +Infinite we must return the last color, but for -Infinite the first one?

Reply:
I think so, yes